src/EFCore.Analyzers/UninitializedDbSetDiagnosticSuppressor.cs

```diff
@@ -0,0 +1,84 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Microsoft.EntityFrameworkCore;
+
+[DiagnosticAnalyzer(LanguageNames.CSharp)]
+public class UninitializedDbSetDiagnosticSuppressor : DiagnosticSuppressor
```

Comment:
nit: I usually like to seal analyzers, codefixes, and suppressors. This is a common pattern in dotnet/roslyn-analyzers as well:

https://github.com/dotnet/roslyn-analyzers/blob/ffd024eb265e97e639a2879d869cf82b5b958992/src/Roslyn.Diagnostics.Analyzers/Core/NamedTypeFullNameNotNullSuppressor.cs#L14


```suggestion
public sealed class UninitializedDbSetDiagnosticSuppressor : DiagnosticSuppressor
```

Reply:
Doing also for InternalUsageDiagnosticAnalyzer